modules/builds-source-code.adoc

```diff
@@ -23,15 +23,15 @@ ifndef::openshift-online[]
   dockerfile: "FROM openshift/ruby-22-centos7\nUSER example" <3>
 endif::[]
 ----
-<1> The `git` field contains the URI to the remote Git repository of the source code. Optionally, specify the `ref` field to check out a specific Git reference. A valid `ref` can be a SHA1 tag or a branch name.
+<1> The `git` field contains the `URI` to the remote Git repository of the source code. Specify the `ref` field to check out a specific Git reference. A valid `ref` can be a SHA1 tag or a branch name, if not specified "master" will be picked as the default value.
```

Comment:
Should this be `main` instead of "master"?

Reply:
No, master is correct here. That is the reason for the documentation update, the code does not automatically detect the main branch.

Reply:
No, it should be `master`. By default builder picks up `master` even for the projects where `main` is the default branch.